src/coreclr/vm/precode.h

```diff
@@ -204,6 +208,35 @@ typedef DPTR(NDirectImportPrecode) PTR_NDirectImportPrecode;
 
 #endif // HAS_NDIRECT_IMPORT_PRECODE
 
+struct InterpreterPrecodeData
```

Comment:
Should this be under `FEATURE_INTERPRETER`?

Reply:
I've just added that commit, it also makes it build for debug / checked only by default.